### DISCUSSION TITLE
Groups doesn't work on non logged device

### DISCUSSION THREAD

**Discussion by DamienAubrun** [rustdesk/rustdesk#7020] [General]
### Bug Description

Hello guys !
I have a problem with rustdesk pro.
I deploy rustdesk on my compagny and i find something strange, when the client is installed on a computer and is not connect to an account, this one can connect on every non logged client. For me if the client is not connect to an account he can't connect to other.
Do you have any idea for this problem ?

i don't know if i was in the good section

thx all ^^

### How to Reproduce

on all non logged client

### Expected Behavior

i want to block the possiblity of all non logged client to connect
Operating system(s) on local side and remote side

### Operating system(s) on local side and remote side

Window 11/10

### RustDesk Version(s) on local side and remote side

1.2.3

### Screenshots

![301104876-17f151dc-e98c-4053-aa6c-9178fd8bcebc](https://github.com/rustdesk/rustdesk/assets/21692172/2aa14e1f-927c-462a-bebb-935ccdc73f7f)

**Comment by rustdesk** [comment]
It is not a bug, we do not have this feature yet. user loggin is required for Group to work, but discussion is welcome.

**Comment by DamienAubrun** [comment]
Oh sorry i don't understand why my post is suppr 
do you think this feature come in future ?
and any idea for block this ?
